src/main/java/gov/cms/madie/models/common/ModelType.java

```diff
@@ -33,6 +35,12 @@ public enum ModelType {
     this.shortValue = shortValue;
   }
 
+  public String getVersionNumber() {
+    Pattern pattern = Pattern.compile("v(\\d+(\\.\\d+)*)");
```

Comment:
Not a big deal, but I saw it once long ago. We could move this compiler to the class level so that it doesn't have to happen on every method call. Also we have to declare it static something like 

```
private static final Pattern VERSION_PATTERN = Pattern.compile("v(\\d+(\\.\\d+)*)"); and then 

public String getVersionNumber() {
    Matcher matcher = VERSION_PATTERN.matcher(this.value);
    return matcher.find() ? matcher.group(1) : null;
  }
```